cmd/liqoctl/cmd/offload.go

```diff
@@ -98,23 +103,28 @@ func newOffloadNamespaceCommand(ctx context.Context, f *factory.Factory) *cobra.
 	options := offload.Options{Factory: f}
 	cmd := &cobra.Command{
 		Use:     "namespace name",
-		Aliases: []string{"ns"},
-		Short:   "Offload a namespace to remote clusters",
+		Aliases: []string{"ns", "namespaces"},
+		Short:   "Offload namespaces to remote clusters",
 		Long:    liqoctlOffloadNamespaceLongHelp,
 
-		Args:              cobra.ExactArgs(1),
-		ValidArgsFunction: completion.Namespaces(ctx, f, 1),
+		ValidArgsFunction: completion.Namespaces(ctx, f, completion.NoLimit),
 
 		PreRun: func(_ *cobra.Command, _ []string) {
 			options.PodOffloadingStrategy = offloadingv1beta1.PodOffloadingStrategyType(podOffloadingStrategy.Value)
 			options.NamespaceMappingStrategy = offloadingv1beta1.NamespaceMappingStrategyType(namespaceMappingStrategy.Value)
 			options.RemoteNamespaceName = remoteNamespaceName
 			options.OutputFormat = outputFormat.Value
+			options.LabelSelector = labelSelector
 			options.Printer.CheckErr(options.ParseClusterSelectors(selectors))
 		},
-
 		Run: func(_ *cobra.Command, args []string) {
-			options.Namespace = args[0]
+			if len(args) == 0 && labelSelector == "" {
+				options.Printer.ExitWithMessage("namespace name or label selector must be specified")
+			}
+			if len(args) != 0 && labelSelector != "" {
+				options.Printer.ExitWithMessage("namespace name and label selector must not be specified together")
+			}
```

Comment:
We need to add an additional check to make sure that no custom remote namespace name is specified when multiple namespaces are targeted for offloading:
```suggestion
			}
			if (len(args) > 1 || labelSelector != "") && remoteNamespaceName != "" {
				options.Printer.ExitWithMessage("the remote namespace name cannot be specified when offloading multiple namespaces at once")
			}
```